rpcs3/Emu/Audio/XAudio2/XAudio2Backend.cpp

```diff
@@ -309,9 +313,12 @@ f64 XAudio2Backend::GetCallbackFrameLen()
 		return _10ms;
 	}
 
+#if _MSC_VER
 	Microsoft::WRL::ComPtr<IXAudio2Extension> xaudio_ext{};
+#endif
 	f64 min_latency{};
 
+#if _MSC_VER
```

Comment:
Maybe we shouldn't build it outside msvc. Though I'm not seeing anything here that should break on other compilers, this is just standard COM.

Reply:
It doesn't link when compiled with GCC or Clang.